setup.py

```diff
@@ -34,7 +34,7 @@
     description="SDE solvers and stochastic adjoint sensitivity analysis in PyTorch.",
     url="https://github.com/google-research/torchsde",
     packages=setuptools.find_packages(exclude=['benchmarks', 'diagnostics', 'examples', 'tests']),
-    install_requires=['torch>=1.6.0', 'numpy==1.19.*', 'boltons>=20.2.1', 'trampoline>=0.1.2', 'scipy==1.5.*'],
+    install_requires=['torch>=1.6.0', 'numpy==1.20.*', 'boltons>=20.2.1', 'trampoline>=0.1.2', 'scipy==1.6.*'],
```

Comment:
Could we change this to >=1.19.0 or something like that? Thanks.

Reply:
Also 'scipy==1.5.*' => `'scipy>=1.5.*'`?

Reply:
Done!